stonesoup/sensor/passive.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+
+from .base import Sensor3DCartesian
+from ..base import Property
+from ..models.measurement.nonlinear import CartesianToElevationBearing
+from ..types.array import CovarianceMatrix
+from ..types.detection import Detection
+
+
+class PassiveElevationBearing(Sensor3DCartesian):
+    """A simple passive sensor that generates measurements of targets, using a
+    :class:`~.CartesianToElevationBearing` model, relative to its position.
+
+    Note
+    ----
+    The current implementation of this class assumes a 3D Cartesian plane.
+
+    """
+
+    ndim_state = Property(
+        int,
+        doc="Number of state dimensions. This is utilised by (and follows in\
+            format) the underlying :class:`~.CartesianToElevationBearing`\
+            model")
+    mapping = Property(
+        [np.array], doc="Mapping between the targets state space and the\
+                        sensors measurement capability")
+    noise_covar = Property(CovarianceMatrix,
+                           doc="The sensor noise covariance matrix. This is \
+                                utilised by (and follow in format) the \
+                                underlying \
+                                :class:`~.CartesianToElevationBearing`\
+                                model")
```

Comment:
There is a mix of different styles being used here and other places which looks messy. Would be good to stick to one standard. Suggest one used for _ndim_state_ and _orientation_ as give you more room for _doc_.